tests/vendor_support_dls.py

```
@@ -0,0 +1,303 @@
+    return index
+
+
+def _modules_used_by_samples() -> set[str]:
+    used: set[str] = set()
+    for sample_yaml in SAMPLES.glob("*.yaml"):
+        used |= set(_ENTITY_TYPE_RE.findall(sample_yaml.read_text()))
+    return used
+
+
+def _git(*args: str) -> str:
+    return subprocess.run(
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)